extensions/container-image/container-image-s2i/deployment/src/main/java/io/quarkus/container/image/s2i/deployment/S2iConfig.java

```diff
@@ -50,6 +51,12 @@
     @ConfigItem(defaultValue = "/home/quarkus/application")
     public String nativeBinaryPath;
 
+    /**
+     * The build timeout.
+     */
+    @ConfigItem(defaultValue = "PT5M")
+    Duration buildTimeout;
```

Comment:
Are we only using binary s2i? For "code" s2i, 5 minutes won't be enough.

Reply:
Only binary at this point